tests/core/utils/test_antimeridian.py

```diff
@@ -199,8 +199,30 @@ def test_antimeridian_multipolygon() -> None:
         (
             Polygon(((170, 40), (170, 42), (190, 42), (190, 40), (170, 40))),
             Polygon(((170, 48), (170, 50), (190, 50), (190, 48), (170, 48))),
-        )
+        ),
     )
     for actual, expected in zip(normalized.geoms, expected.geoms):
         assert actual.exterior.is_ccw
         assert actual.equals(expected), f"actual={actual}, expected={expected}"
+
+
+def test_antimeridian_enclose_poles() -> None:
```

Comment:
Could you add a test that ensures we hit both of the branches at https://github.com/stac-utils/stactools/pull/416/files#diff-39c4b9c63d34aa4ac3f9e5916273757b58071aeca592e256e8306f41729f0499R269, starting with

```
if start[0] > 0:
```
?

Reply:
We already hit both of those branches -- one of them is the north pole enclosure (starting at +170° longitude) and the other is the south pole enclosure (starting at -170° longitude). From a local coverage run:
![Screenshot 2023-04-10 at 08-09-54 Coverage for src_stactools_core_utils_antimeridian py 97%](https://user-images.githubusercontent.com/58314/230918213-7594a4c0-8dc4-4bc5-b4e9-d1e65e528baa.png)

Reply:
Ah right, I forgot that a single polygon could cross both. Thanks!